docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DockerCompose.java

```diff
@@ -44,6 +45,6 @@ static Version version() throws IOException, InterruptedException {
     Optional<String> id(Container container) throws IOException, InterruptedException;
     String config() throws IOException, InterruptedException;
     List<String> services() throws IOException, InterruptedException;
-    boolean writeLogs(String container, OutputStream output) throws IOException;
+    void writeLogs(String container, OutputStream output) throws IOException, InterruptedException;
```

Comment:
I think this is technically a breaking change - can we return a boolean still and not propagate the interrupted exception?